frontend/src/app/plot/plot.component.ts

```diff
@@ -113,6 +113,21 @@ export class PlotComponent implements OnInit {
             message[i].timestamp_diffs[0], // TS Diff id.
             this.plot_data.length - 1
           );
+
+          // Plot the latencies if they are present in the sample.
```

Comment:
Do we expect multiple latency per sample? Why use of latencies as opposed to latency?

Reply:
My thinking was that a single point is 'latency' and so the array of that would be 'latencies'. Similar convention to how I named it 'timestamps' instead of just 'timestamp'